### PR TITLE
V6.0.0 Release

### PR DESCRIPTION
**Pull request by VladislavFitz**
Brand new InstantSearch Core Swift v2 release

### CODE REVIEW

**Comment by VladislavFitz** [comment]
@BenoitPerrot I updated the [license file](https://www.github.com/algolia/instantsearch-core-swift/blob/instantsearch-v2/LICENSE). Could you check if it is good?

**Comment by VladislavFitz** [comment]
@robertmogos Bitrise checked passed, ready for release 👌

File: InstantSearchCore.xcodeproj/xcshareddata/xcschemes/InstantSearchCore-iOS.xcscheme [hunk 1]
```diff
@@ -71,6 +71,18 @@
...
       </MacroExpansion>
+      <EnvironmentVariables>
+         <EnvironmentVariable
+            key = "ALGOLIA_APPLICATION_ID"
+            value = "1M1U6ZWKZP"
+            isEnabled = "YES">
+         </EnvironmentVariable>
+         <EnvironmentVariable
+            key = "ALGOLIA_API_KEY"
+            value = "c762c14ebbd970c7f5c7ec6654b26472"
```
Comment (robertmogos) [inline InstantSearchCore.xcodeproj/xcshareddata/xcschemes/InstantSearchCore-iOS.xcscheme]: Shouldn't this be removed? As well as `ALGOLIA_APPLICATION_ID`

Comment (VladislavFitz) [reply InstantSearchCore.xcodeproj/xcshareddata/xcschemes/InstantSearchCore-iOS.xcscheme]: My bad, it leaked at some point.

**Pull request merged**